low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -194,6 +194,7 @@ SYSTEM_INFORMATION:
    sysfs_base_path: /sys/
    sspl_state: active
    global_config_copy_url: ""
+   server_fru_list: ['disk', 'psu', 'fan']
```

Comment:
> call it server_declare_fru
or sspl_supported_server_frus would be better?

> lets keep disk & psu in this list, where we know for sure
then for 'fan' sensor, it will give fru:false, as ipmitool fru list cmd not giving fan fru info